cryptlex/cryptlex-enterprise/values.yaml

```diff
@@ -250,6 +250,8 @@ webApi:
   # Postmark
   postmark:
     accountToken: ""
+  googleAnalytics:
+    key: ""  # G-XXXXXXX
```

Comment:
keep comments above the property and move this to google section

Reply:
@adnan-kamili we use recaptcha and analyticskey only in web-api, so is making a seperate google section necessary at this point ?

Reply:
Third party config is usually grouped unded third party provider name so:

 # Google
  google:
    recaptcha:
       secretKey: ""
    analytics:
       key: ""